Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -8,76 +8,152 @@
+                                Image("github-logo")
+                                    .resizable()
+                                    .aspectRatio(contentMode: .fit)
+                                    .frame(width: 18, height: 18)
+                                    .padding(.leading, 2)
+                                
+                                Text("GitHubRepo")
+                                    .frame(maxWidth: .infinity, alignment: .leading)
+                            }
+                        }
                     }
-                } header: {
-                    Text("Vehicles")
-                }
-                
-                Section {
-                    NavigationLink {
-                        AuthenticationView(viewModel: authenticationViewModel)
-                    } label: {
-                        Label {
-                            Text("Profile")
-                        } icon: {
-                            Image(systemName: "person")
+                    
+                    
+                    
+                    Section {
+                        ForEach(viewModel.vehicles) { vehicle in
+                            VStack {
+                                Text(vehicle.name)
+                                    .font(.headline)
+                                
+                                Text(vehicle.make)
+                                
+                                Text(vehicle.model)
+                            }
                         }
+                        
+                        Button {
+                            isShowingAddVehicle.toggle()
+                        } label: {
+                            Text("Add Vehicle")
+                        }
+                    } header: {
+                        Text("Vehicles")
                     }
+                    
+                    Section {
+                        NavigationLink {
+                            AuthenticationView(viewModel: authenticationViewModel)
+                        } label: {
+                            Label {
+                                Text("Profile")
+                            } icon: {
+                                Image(systemName: "person")
+                            }
+                        }
+                    }
+                    
+                    VStack(alignment: .center) {
+                        HStack(spacing: 0) {
+                            Text("❤️ ")
+                            Text(infoViewModel.message)
+                            Text(" ❤️")
+                        }
+                            .font(.title3)
+                            .foregroundColor(.blue)
+                        
+                        Text(infoViewModel.title)
+                            .font(.title)
+                            .bold()
+                        
+                        if let appVersion = infoViewModel.appVersion {
```

Comment:
Adding the app version number is not in the scope of this issue and already is there